### PR TITLE
fix: Hide app badges on Eigen

### PR DESCRIPTION
**Pull request by rajsam003**
This PR fixes the bug mentioned on this ticket [CX-2125](https://artsyproduct.atlassian.net/browse/CX-2125)

* Congratulation message will only be shown on the Order submitted page.
* App Badges will not be displayed in Eigen.

### CODE REVIEW

File: src/v2/Apps/Order/Components/TransactionDetailsSummaryItem.tsx [hunk 1]
```diff
@@ -318,7 +325,6 @@ export const TransactionDetailsSummaryItemFragmentContainer = createFragmentCont
             }
           }
         }
-        state
```
Comment (olerichter00) [inline src/v2/Apps/Order/Components/TransactionDetailsSummaryItem.tsx:321]: 👀

**Review by olerichter00 — APPROVED** [review]
LGTM 🚀

**Review by damassi — APPROVED** [review]
We should update corresponding tests before merge, but code looks good! 👍

File: src/v2/Apps/Order/Components/TransactionDetailsSummaryItem.tsx (hunk 1 above)
Comment (rajsam003) [reply src/v2/Apps/Order/Components/TransactionDetailsSummaryItem.tsx:321]: We don't need this here.

**Review by olerichter00 — APPROVED** [review]

**Pull request merged**